<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QReader</class>
 <widget class="QWidget" name="QReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>617</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="display">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Detected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDone">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>500</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
